common/v2/database/v1.0.0/generateDefaultValues.ts

```diff
@@ -0,0 +1,185 @@
+import * as R from 'ramda';
+import { generateUUID, generateAssetUUID, generateContractUUID } from 'v2/utils';
+import { Fiats, DEFAULT_ASSET_DECIMAL } from 'v2/config';
+import { NODES_CONFIG, NETWORKS_CONFIG } from '../data';
+import {
+  Asset,
+  ExtendedAsset,
+  ExtendedContract,
+  LocalStorage,
+  NetworkId,
+  NetworkLegacy,
+  WalletId,
+  Network,
+  TSymbol,
+  Fiat,
+  ContractLegacy,
+  AssetLegacy,
+  LSKeys
+} from 'v2/types';
+
+import { SeedData, StoreAction } from './types';
+import { toArray, toObject, add } from './helpers';
+/* Transducers */
+const addNetworks = add(LSKeys.NETWORKS)((networks: SeedData) => {
+  const formatNetwork = (n: NetworkLegacy): Network => {
+    const baseAssetUuid = generateUUID();
```

Comment:
This should be deterministic.

Reply:
changed to `generateAssetUUID(n.chainId)`